.gitmodules

```diff
@@ -1,13 +1,3 @@
 [submodule "src/tools/firrtl"]
 	path = src/tools/firrtl
 	url = https://github.com/ucb-bar/firrtl.git
-[submodule "src/tools/pyyaml"]
```

Comment:
Hmm. While I'm not opposed to using pip and the other standard tools, I do want to make sure that this can continue to work out of the box on rusty CAD machines since that is the type of environment where Hammer will run. Maybe we can move these repos to `contrib/` (preferred) or `vending/` or some other prefix which we can use if Hammer is cloned onto an old CAD machine?

Reply:
Oh, I understand. I have test this on CentOS 6 and CentOS 7 to make it work in the old environment.
If you would like to use `contrib/` to vend these libraries, how about I give 2 flows, one for vending, another for `pip` in 2 different `sourceme.sh`?

Reply:
That sounds reasonable - maybe let's try that and see what happens.

Reply:
OK, working on it now :)